plasmapy/formulary/collisions.py

```diff
@@ -2196,10 +2240,11 @@ def __init__(
         # Note: This function uses CGS units internally to coincide with our references.
         # Input is taken in MKS units and then converted as necessary. Output is in MKS units.
 
+        # _v_a and _T_a denote attributes only to be used in cases in which a conversion between
+        # temperature and velocity is acceptable based on the assumptions of a collision frequency
+
         if v_a is None:
-            if T_a is not None:
-                v_a = thermal_speed(T_a, test_particle)
-            else:
+            if T_a is None:
                 raise ValueError("Please specify either v_a or T_a.")
         elif T_a is not None:
```

Comment:
I think we should change it so both can be specified -as you said we'll need that for v_ss' and more generally for any case of a drifting Maxwellian